rewrites.js

```diff
@@ -122,10 +122,6 @@ exports.REWRITES = [
     source: '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:slug/admin/:section?',
     destination: '/admin-panel',
   },
-  {
-    source: '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:collectiveSlug/contact',
-    destination: '/collective-contact',
```

Comment:
:no_entry_sign: We don't know whether this URL is used and shared by collectives outside of the platform, let's not make this a breaking change. If we really want to remove this page (I personally don't think we should) then we'll need to deploy the first version of the modal first and check in the logs that the contact page is not accessed.

Reply:
Thanks. I've added back the contact form page. One other thing I was thinking is only to show the two buttons (in the hero section and below the teams section) only when the user is logged in. Since otherwise we might introduce more spam? Also the contact from we currently have do not allow users to submit it unless they are logged in. So if the user isn't logged in we only show the contact button in the navbar; 

![image](https://user-images.githubusercontent.com/12435965/144651043-bda62485-2382-4ba4-b807-615bb4dfa321.png)

When clicked on this they will require to log in and afterwards redirected to the contact form. My thinking is the above button (as well as when you click on Actions -> Contact) should direct the user to the contact page and only the two buttons should show the modal (if the user is logged in). This way we make sure all users whether they are logged in or not can access the contact page.

Reply:
> One other thing I was thinking is only to show the two buttons (in the hero section and below the teams section) only when the user is logged in 

Good question. Maybe ask Alanna and Alina on Slack, I think they're to most used to dealing with this contact issue